src/fheroes2/campaign/campaign_scenariodata.cpp

```diff
@@ -700,7 +700,7 @@ namespace Campaign
             return description;
         }
         case ScenarioBonusData::RESOURCES: {
-            std::string description( _amount > 0 ? _( "The kingdom will have by %{amount} more %{resource} at the start of the scenario." )
+            std::string description( _amount > 0 ? _( "The kingdom will receive by %{amount} more %{resource} at the start of the scenario." )
```

Comment:
The problem here and elsewhere is that "by" is not correct in this nor in case there is less.  Also "additional" is better than "more" in this case:
"The kingdom will receive 500 additional gold at the start of the scenario." This works if the resource is also one unit.

```suggestion
            std::string description( _amount > 0 ? _( "The kingdom will receive by %{amount} more %{resource} at the start of the scenario." )
```